demos/virtual_ai_assistant_demo/main.py

```diff
@@ -210,8 +210,16 @@ def load_context(file_paths: List[str]) -> None:
                                           memory=memory, node_postprocessors=[ov_reranker])
 
 
+# this is necessary for thinking models e.g. deepseek
+def emphasize_thinking_mode(token: str) -> str:
```

Comment:
Hi @adrianboguszewski may I understanding why this function is needed ?

Reply:
To make thinking different from the real answer. Please run it and you will see the difference ;)

Reply:
OK i got you. its for decoration only, right ?

Reply:
Correct